docs/index.adoc

```diff
@@ -125,22 +125,91 @@ This layered standard defines additional capability flags in the layered standar
 | `Layered standard based on FMI 2.0 and FMI 3.0 for describing and implementing XCP support for FMUs, which can either provide an XCP service or allow direct memory access via A2L files.`
 | String with a brief description of the layered standard that is suitable for display to users.
 
-|`containsXcpService`
+|====
+
+The root element must contain at least one, and can contain more than one `Interface` elements.
+Each such element specifies one set of interface information for calibration using one underlying variable description file (A2L).
+Multiple elements might be provided if there are different XCP slaves embedded in the FMU, or there are other reasons why multiple A2L variable description files that apply in parallel have to be provided.
+
+Each `Interface` element has the <<table-schema-fmi-ls-xcp-interface-attributes,following attributes>>:
+
+.Interface Element Attribute Details.
+[[table-schema-fmi-ls-xcp-interface-attributes]]
+[cols="1,1,1,2",options="header"]
+|====
+|Attribute
+|Namespace

```

Comment:
Why is the namespace column empty?

Reply:
Because the attributes are not in any namespace